v2/client.go

```diff
@@ -20,6 +20,11 @@ import (
 	"github.com/influxdata/influxdb1-client/models"
 )
 
+const (
+	MIMETypeJSON = "application/json"
```

Comment:
If we go with the `QueryAsRaw` idea, then these can stay. Otherwise, these should be lowercase so the constants are unexposed and not part of the API.

Reply:
For sure, I agree. This was probably an oversight when making the pull request. Thank you for catching it!